index.html

```diff
@@ -1,344 +1,267 @@
-            <div class="link-container">
-              <img src="./assets/Logo-design.png" alt="Logo design" />
-              <div class="link-content">
-                <img src="./assets/Icon-1.png" alt="Logo design icon" />
-                <span>Logo design</span>
-                <p>
-                  Request a quote
-                  <img src="./assets/Arrow.png" alt="white-arrow" />
-                </p>
-              </div>
-            </div>
-            <div class="link-container">
-              <img src="./assets/Development.png" alt="Development" />
-              <div class="link-content">
-                <img src="./assets/Icon-2.png" alt="Development icon" />
-                <span>Development</span>
-                <p>
-                  Request a quote
-                  <img src="./assets/Arrow.png" alt="white-arrow" />
-                </p>
-              </div>
-            </div>
-            <div class="link-container">
-              <img src="./assets/Uiux.png" alt="UI/UX Design" />
-              <div class="link-content">
-                <img src="./assets/Icon-3.png" alt="UI/UX design icon" />
-                <span>UI/UX Design</span>
-                <p>
-                  Request a quote
-                  <img src="./assets/Arrow.png" alt="white-arrow" />
-                </p>
-              </div>
-            </div>
-            <div class="link-container">
-              <img src="./assets/Digital.png" alt="Digital Margeting" />
-              <div class="link-content">
-                <img src="./assets/Icon-4.png" alt="Deigital Marketing icon" />
-                <span>Digital Marketing</span>
-                <p>
-                  Request a quote
-                  <img src="./assets/Arrow.png" alt="white-arrow" />
-                </p>
-              </div>
-            </div>
-            <div class="link-container">
-              <img src="./assets/Content.png" alt="Content Writing" />
-              <div class="link-content">
-                <img src="./assets/Icon-5.png" alt="Content Writing icon" />
-                <span>Content Writing</span>
-                <p>
-                  Request a quote
-                  <img src="./assets/Arrow.png" alt="white-arrow" />
-                </p>
-              </div>
-            </div>
-            <div class="link-container">
-              <img src="./assets/Email.png" alt="Email Marketing" />
-              <div class="link-content">
-                <img src="./assets/Icon-6.png" alt="Email Marketing icon" />
-                <span>Email Marketing</span>
-                <p>
-                  Request a quote
-                  <img src="./assets/Arrow.png" alt="white-arrow" />
-                </p>
-              </div>
+      </div>
+    </section>
+    <section class="section3-container">
+      <h2>Our <span>Services</span></h2>
+      <div class="wrapper">
+        <div class="link-wrapper">
+          <div class="link-container-1">
```

Comment:
Like this
`<div class="link-container link-container-1">`

Reply:
I changed them as you said thanks